main.go

```diff
@@ -113,19 +113,20 @@ func main() {
 
 	root.AddCommand(namespace.Namespace(ctx))
 	root.AddCommand(cmd.Init())
-	root.AddCommand(up.Up())
-	root.AddCommand(cmd.Down())
-	root.AddCommand(cmd.Status())
-	root.AddCommand(cmd.Doctor())
-	root.AddCommand(cmd.Exec())
+
+	root.AddCommand(cmd.Dev())
+
 	root.AddCommand(preview.Preview(ctx))
-	root.AddCommand(cmd.Restart())
 	root.AddCommand(cmd.UpdateDeprecated())
 	root.AddCommand(deploy.Deploy(ctx))
 	root.AddCommand(destroy.Destroy(ctx))
 	root.AddCommand(deploy.Endpoints(ctx))
 	root.AddCommand(generateFigSpec.NewCmdGenFigSpec())
 
+	// Aliases. They are already under `dev`
+	root.AddCommand(up.Up())
+	root.AddCommand(cmd.Down())
+
```

Comment:
I couldn't find a better way for keeping this as aliases. I'm open for ideas!

Reply:
That's how we did it with `kubeconfig` command.
We have `okteto kubeconfig` code in main and `okteto context update-kubeconfig`(deprecated) code in context package